weave-js/src/components/Icon/Icon.tsx

```diff
@@ -140,7 +142,7 @@ import {ReactComponent as ImportRocketLaunch} from '../../assets/icons/icon-rock
 import {ReactComponent as ImportRowHeightLarge} from '../../assets/icons/icon-row-height-large.svg';
 import {ReactComponent as ImportRowHeightMedium} from '../../assets/icons/icon-row-height-medium.svg';
 import {ReactComponent as ImportRowHeightSmall} from '../../assets/icons/icon-row-height-small.svg';
-import {ReactComponent as ImportRowHeightXLarge} from '../../assets/icons/icon-row-height-xlarge.svg';
+import {ReactComponent as ImportRowHeightXlarge} from '../../assets/icons/icon-row-height-xlarge.svg';
```

Comment:
This change is a result of addicon's default name translation rules. If we don't like it, we can modify that script to have a special case for this icon. (Only current naming exception is VertexGCP.)